packages/@react-native/tester/package.json

```diff
@@ -27,7 +27,7 @@
     "e2e-test-ios": "./scripts/maestro-test-ios.sh"
   },
   "dependencies": {
-    "@react-native/popup-menu-android": "0.77.0-nightly-20241001-223e98cc4",
+    "@react-native/popup-menu-android": "0.78.0-nightly-20250113-d4407d6f7",
```

Comment:
I've seen this get bumped when running the rn-script recently, possibly just a integration or merge issue? Will keep on eye on it in integrations to make sure it's being bumped correctly